config/inputs/house/household_heating_technology.yml

```diff
@@ -12,7 +12,7 @@ options:
       households_water_heater_combined_network_gas_share: 100
       households_water_heater_heatpump_ground_water_electricity_share: 0
       households_water_heater_micro_chp_network_gas_share: 0
-      households_space_heater_heatpump_add_on_electricity_share: 0
+      households_space_heater_heatpump_air_water_electricity: 0
```

Comment:
Should this not be "[households_space_heater_heatpump_air_water_electricity**_share**](https://github.com/quintel/etsource/blob/6acb69f3da057fdc254d8bdc7ec379828e211b29/inputs/demand/households/households_heating/households_space_heater_heatpump_air_water_electricity_share.ad)"?